pocs/filterwheel/libefw.py

```diff
@@ -0,0 +1,181 @@
+import ctypes
+import enum
+
+from pocs.camera.sdk import AbstractSDKDriver
+from pocs.utils import error
+from pocs.utils.library import load_library
+
+
+class EFWDriver(AbstractSDKDriver):
+    # Because ZWO EFW library isn't linked properly have to manually load libudev
+    # in global mode first, otherwise get undefined symbol errors.
+    _libudev = load_library('udev', mode=ctypes.RTLD_GLOBAL)
+
+    def __init__(self, library_path=None, **kwargs):
+        """Main class representing the ZWO EFW library interface.
+
+        On construction loads the shared object/dynamically linked version of the EFW SDK library,
+        which must be already installed (see https://astronomy-imaging-camera.com/software-drivers).
+
+        The name and location of the shared library can be manually specified with the library_path
+        argument, otherwise the ctypes.util.find_library function will be used to try to locate it.
+
+        Args:
+            library_path (str, optional): path to the libary e.g. '/usr/local/lib/libEFWFilter.so'
```

Comment:
```suggestion
            library_path (str, optional): path to the library e.g. '/usr/local/lib/libEFWFilter.so'
```

Lots of' those `libary` copy/paste errors. :)